packages/components/src/components/Checkbox/index.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { PureComponent } from 'react';
+import React, { FunctionComponent } from 'react';
```

Comment:
you don't need to import FunctionComponent, just use regular functions. eg `const CheckBox = ({ isChecked, children, onClick, ...rest }: Props) => {...}`

some reasoning here https://github.com/trezor/trezor-suite/pull/436#discussion_r317622482